### DISCUSSION TITLE
Error: Bad Input to API call

### DISCUSSION THREAD

**Discussion by teeetw0** [samschott/maestral#686] [Q&A]
I'm running Maestral 1.6.2 under macOS 12.4. Since a few days I'm getting an error message:

"Bad Input to API call"
{"error":"v1_retired"}

<img width="277" alt="Screen Shot 2022-05-31 at 12 07 01" src="https://user-images.githubusercontent.com/106627468/171290533-a923e5db-401e-48d0-97cb-249233c2b9a3.png">

After that happens, Maestral pauses syncing. I can resume syncing, but the error message keeps popping back up. 

Getting this error on two Macs (iMac and MBP). Tried unlinking and re-linking Dropbox account on both machines but the error keeps coming back.

Would be great to get this fixed. Can provide more info if needed.

**Comment by JoePunkt** [comment]
I have the same Problem

**Comment by dkrea** [comment]
This is weird since Dropbox apparently retired the v1 api in 2017, unless this is for a seperate api?
https://dropbox.tech/developers/api-v1-shutdown-details

**Comment by doersino** [comment]
This problem is being tracked in issue #683 - it's a bug on Dropbox's side, but Maestral could handle it more gracefully.

**Comment by boorgarry** [comment]
I have the same Error,

![20220603 Maestral Fehler](https://user-images.githubusercontent.com/93041948/171916728-71126188-8460-49b7-a362-888672753cea.jpg)
